test/CodeGenILTests/ReplicationTests.cs

```diff
@@ -571,5 +571,31 @@ public void MSIL_FuncCall_Double_SomeGuides()
                 },
                 output["test4"]));
         }
+
+        [Test]
+        [Category("Failure")] // A DSCore.Color object fails with a "Unable to locate managed object for given dsObject." exception
```

Comment:
Does this work now after [Tibi's fix](https://github.com/DynamoDS/Dynamo/commit/7b0ccf9da6d484e2d1b25124d8e32b0990dfd9f6)?

Reply:
Yes, I will remove this test.